packages/react-instantsearch/src/connectors/hierarchicalMenuLogic.js

```diff
@@ -0,0 +1,229 @@
+import PropTypes from 'prop-types';
+import { SearchParameters } from 'algoliasearch-helper';
+import {
+  cleanUpValue,
+  getCurrentRefinementValue,
+  getIndex,
+  getResults,
+  refineValue,
+} from '../core/indexUtils';
+
+export const getId = props => props.attributes[0];
```

Comment:
I don't really understand this file, can it be reused in `connectHierarchicalMenu` or is that a stupid idea?